dotcom-rendering/cypress/plugins/index.js

```diff
@@ -7,16 +7,41 @@
 // You can read more here:
 // https://on.cypress.io/plugins-guide
 // ***********************************************************
+import path from 'path';
 
 const webpackPreprocessor = require('cypress-webpack-preprocessor-v5');
 
 module.exports = (on, config) => {
 	config.env = { ...config.env, ...process.env };
 
 	const webpackConfig = webpackPreprocessor.defaultOptions;
-	webpackConfig.webpackOptions.module.rules[0].exclude =
+	// Adding this here so that we can import the fixture in a cypress test
+	webpackConfig.webpackOptions.resolve = {
+		extensions: ['.ts', '.js'],
+		alias: {
+			Standard$: path.resolve(
```

Comment:
Would this end up aliasing any import which ends in `Standard`? If so I wonder if we'd benefit from being more explicit, making the alias `generated-fixtures/Standard` then doing the import directly (e.g `import { Standard } from 'generated-fixtures/Standard';`), to avoid clashes?

Reply:
I don't think we actually need the alias on reflection, so I have removed it

Reply:
Taking another look at this, I don;t think we need to use the alias field, so I've taken it out